docs/src/styles/globals.css

```diff
@@ -37,3 +37,22 @@ a {
   font-weight: 400;
   font-style: normal;
 }
+
+.DocSearch-Form {
```

Comment:
Is there any reason why these styles in global CSS but not in component styles?

Reply:
Yes. Because DocSearch appends modal directly in body. 
So I couldn't set styles anywhere else.